### PR TITLE
What are you doing mateo

### PR DESCRIPTION
**Pull request by Raphael-ML-Picard**
Oui oui
Some name to it

### CODE REVIEW

**Comment by Raphael-ML-Picard** [comment]
@Paijmans Can you please review my beautiful code?

File: release_notes.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Release notes
 
+## v0.0.3 - In progress
+
+- Added the `cone` functionality to support data export
```
Comment (Paijmans) [inline release_notes.md:5]: This is amazing! Compliments.

File: sequioa/crown.md [hunk 2]
```diff
@@ -10,3 +10,13 @@ Provide the support and connection between the higher level sub-modules and the
 
 Display green colour.
 Handle chemical data conversion and output.
+
+
+## Cones
+
+Allow the user to export data from the application.
+TODO: Secure the exported file
```
Comment (Paijmans) [inline sequioa/crown.md:18]: Please change the shape of the leaves!

File: sequioa/trunk.md [hunk 3]
```diff
@@ -11,6 +11,10 @@ The layer that protects our application from external threats.
 
 Greatly improve resistance to overheating in fire situation.
 
-### Redcuded sap content
+### Reduced sap content
 
 Keeping the `bark` dry means we leave less opporunities for things to catch on fire.
+
+### Increase moss volume
+
```
Comment (Paijmans) [inline sequioa/trunk.md:19]: Please add water storage and find out a good name.

**Review by Paijmans — CHANGES_REQUESTED** [review]
- Good name for water storage.
- New leave shape

File: release_notes.md (hunk 1 above)
Comment (Raphael-ML-Picard) [reply release_notes.md:5]: Thanks you Dr. Paijmans

**Comment by mateohv** [comment]
I have no idea what I am doing

**Comment by bkarpati** [comment]
Thanks for participating in the Git workshop. I'm going to close this PR for now and also delete the remote branch.
Please stay tuned for the follow-up.

**Pull request closed without merging**